README.md

```diff
@@ -1 +1,62 @@
-# Ylitse mobile application
+<h1 align="center"> Ylites Mobile App</h1> <br>
+<p align="center">
+    <img alt="Ylitse" title="Ylitse" src="https://github.com/ylitse/ylitse-app/blob/master/ylitse_logo.png">
+</p>
+
+<p align="center">
+  <a href="#">
+    <img alt="Download on the App Store" title="App Store" src="http://i.imgur.com/0n2zqHD.png" width="140">
+  </a>
+
+  <a href="#">
+    <img alt="Get it on Google Play" title="Google Play" src="http://i.imgur.com/mtGRPuM.png" width="140">
+  </a>
+</p>
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Features](#features)
+- [Build Process](#build-process)
+- [Sponsors](#sponsors)
+- [Acknowledgments](#acknowledgments)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg?style=flat-square)](http://commitizen.github.io/cz-cli/)
+
+Ylitse  connects young parents from a child protection background (mentees) to mentors from the same background who have the time to help and have been trained on how to. The Ylitse App supports this process, by allowing the mentees to find suitable mentors, connect with them and get chat advice anonymously.
+
+**Available for both iOS and Android.**
+
+<p align="center">
+  <img src="https://github.com/ylitse/ylitse-app/blob/master/main_screen.png" width=350>
+</p>
+
+## Features
+
+* Users and mentors can sign up and log in to the app.
+* Browse through the list of mentors with their introductions text and supported languages.
+* Users can initiate and continue chat conversations with mentors.
+
+## Build Process
+
+- Follow the [React Native Guide](https://facebook.github.io/react-native/docs/getting-started.html) for getting started building a project with native code. **A Mac is required if you wish to develop for iOS.**
+- Clone or download the repo
+- `npm i` to install dependencies
+- `npm start` to link react-native dependencies
+- `npm run ios` to start the packager and run the app in the iOS simulator
+- `npm run android` to start the packager and run the app in the the Android device/emulator
+
+## Sponsors
+
+Ylitse app is proudly sponsored by [SOS Lapsikylä](https://www.sos-lapsikyla.fi/).
+
+## Acknowledgments
+
+TODO: update content.
```

Comment:
There's at least Azure, but we are not using it yet, so probably not adding here for now.

Besides that, what else would make sense? A list of major os libs that we are using? (react, react-native, etc ...)?

Reply:
https://gitlab.com/ylitse/ylitse-mobile Ivan had written some libs on old README.

Reply:
But not sure at all if it makes sense or not.

Reply:
At some point it would make sense to talk about ylitse-api but maybe it could be added later?